src/components/victory-bar/bar.jsx

```diff
@@ -35,26 +36,19 @@ export default class Bar extends React.Component {
       this.getHorizontalBarPath(position, width) : this.getVerticalBarPath(position, width);
   }
 
-  renderBar(props) {
-    const style = Helpers.evaluateStyle(props.style, props.datum);
+  render() {
+    const style = Helpers.evaluateStyle(this.props.style, this.props.datum);
```

Comment:
Consider destructuring a ref to `props`: `const { props } = this;` since it's being used repeatedly.